var/spack/repos/builtin/packages/environment-modules/package.py

```diff
@@ -3,18 +3,29 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
 import os
 
 
 class EnvironmentModules(Package):
     """The Environment Modules package provides for the dynamic
     modification of a user's environment via modulefiles."""
 
-    homepage = "https://sourceforge.net/p/modules/wiki/Home/"
-    url = "http://prdownloads.sourceforge.net/modules/modules-3.2.10.tar.gz"
+    homepage = 'https://modules.readthedocs.io/en/latest/'
```

Comment:
I would suggest to use here a link to the website (`https://cea-hpc.github.io/modules/`) rather a link to the documentation portal.